src/ncp/changed_props_set.cpp

```diff
@@ -83,6 +83,11 @@ const ChangedPropsSet::Entry ChangedPropsSet::mSupportedProps[] = {
 #if OPENTHREAD_ENABLE_CHANNEL_MANAGER
     {SPINEL_PROP_CHANNEL_MANAGER_NEW_CHANNEL, SPINEL_STATUS_OK, true}, // 29
 #endif
+#if OPENTHREAD_ENABLE_JOINER
+    {SPINEL_PROP_LAST_STATUS, SPINEL_STATUS_JOIN_SECURITY, false},    // 30
+    {SPINEL_PROP_LAST_STATUS, SPINEL_STATUS_JOIN_RSP_TIMEOUT, false}, // 31
+    {SPINEL_PROP_LAST_STATUS, SPINEL_STATUS_JOIN_SUCCESS, false},     // 32
```

Comment:
Need to add `JOIN_NO_PEERS` here?

Also, we have reached beyond 32 items... Current code uses `uint32_t` to track the changes...
Submitted https://github.com/openthread/openthread/pull/2883 to increase the max number of entries.

Reply:
will update after the depending PRs are merged. Thanks.